node/handlers/catalog.ts

```diff
@@ -32,6 +33,7 @@ export const catalog = async (ctx: Context) => {
       'Authorization': authToken,
       'Proxy-Authorization': authToken,
       'User-Agent': process.env.VTEX_APP_ID,
+      VtexIdclientAutCookie: authToken,
```

Comment:
Wow! That's not safe at all - you're adding this app's token to communicate with an API, yet we have no idea what permissions the actual caller had. Why did you have to send this header? This *might* effectively make a private API public, through this proxy, if this app has sufficient permissions.